tensorboard/uploader/proto/export_service.proto

```diff
@@ -71,38 +72,6 @@ message StreamExperimentsResponse {
   repeated Experiment experiments = 2;
 }
 
-// Metadata about an experiment.
-message Experiment {
```

Comment:
This is a breaking change, and tests are properly catching it. Even if
we fixed the code in this change, the internal code would break at sync
time.

If you want to centralize both services on a common `Experiment` and
mask, that will be a trickier change.